packages/common/src/repository/categoryRepository.ts

```diff
@@ -0,0 +1,41 @@
+import { SelectedFields } from 'drizzle-orm/pg-core/query-builders/select.types';
+import { SQL } from 'drizzle-orm/sql';
+import { isEmpty } from 'lodash-es';
+
+import { GlobalConfig } from '../config/config.js';
+import { Category, NewCategory, dictionaryCategories } from '../models/dictionary_categories.js';
+
+export default class CategoryRepository extends GlobalConfig {
```

Comment:
same comment about classes vs functional composition as before.

looks like what you wanted to do here was something along the lines of

```js
const something = (opts) => {
  const save = async () => {};
  const select = async () => {};

  return {
    save,
    select,
  };
```
...which works great if you want to pass initialisation options to the main function.

otherwise, what you're trying to do is just something like this:

```js
const something = {
  save: async () => {},
  select: async () => {},
};
```

unsure on which one you're aiming for though.